skycultures/ticuna/description.en.utf8

```diff
@@ -0,0 +1,22 @@
+<h3 align="center">The indigenous testimonies</h3>
+<p align="justify">“The stars fit into their respective constellations. First is Wücütcha. After winter, summer begins to walk. It’s the Jaguar. After him comes Coyathicüra in the month of May, at nightfall, when the Cayman’s Jaw appears on the beach. Baweta appears in September, the month of the turtle. In May, Wücütcha and Coyatchicüra fall. All fall together at the end of May. In July, they come again. Baweta already begins to appear in July. When Wücütcha falls, beginning in May, Caymans and turtles begin to appear. The apex is in September”.</p>
+<p align="justify">“In May, all the constellations fall, together with their planets. It’s summertime. Later, in July, they appear again for the New Year. Coya and Wücütcha leave the sky from May to September. Later, in July, they appear again.” (Fernando Marques or Ütchancu – translated by Lucho)</p>
+<p align="justify">According to the explanations found in Enepü (a Ticuna community in the indigenous Land Eware II, corresponding to the Brazilian State of Amazon, in the Western Amazonia), the influence of seasonality upon Ticuna subsistence activities is more marked along the Solimões given that in the headwaters of the igarapés, where Enepü is situated, plenty exists all year long. In Enepü, trees such as the ingá, and palms such as the açaí are fertile all year long due to the “virginity” and the enrichment of the soil by natural compost. The same thing occurs with the fishing in Lake Preto within the limits of Enepü, where people living are able to supply themselves abundantly with fish all year long. On the Solimões, however, one finds fish only from April to July, when the water level drops. Abundance in the hunt is also from April to July/September along the Solimões. Fishing becomes more difficult when the river is up. The fruit of the native açaí palm is collected in summer, and customarily, saved for preparation during winter, the time of poor hunting and fishing. Through planning and regular gardening, agricultural harvests are available all year long. The harvest of cultivated plants also occurs during the drought months. A certain degree of flexibility exists in the adaptation of the agricultural and extractive calendar to the rainy season, when fishing and hunting are not productive, and the dry season, when fishing is more abundant but the difficulties involved in navigation and transport grow as the water level along the igarapés linking the river drops. Traditional culture is not prepared, however, for unforeseen alterations in the rainy season such as it occurred in 2002. In that year, July saw the beginning of the repiquete (the rise on water levels), which normally occurs in September. This destroyed the short-range agricultural cycle and made the fish scarcer, changing alimentary habits.</p>
```

Comment:
For a region so close to the equator, I think it would be helpful to add the month to mentions of "summer" and "winter". I read about abundance in the hunt April-July (which is commonly regarded "winter" in the southern hemisphere), but the next sentence speaks of winter as time of poor hunting. This is contradictory.

Reply:
Please remove align="justify" and align="center" parts